baram/ec2_manager.py

```diff
@@ -1,90 +1,141 @@
 import boto3
 
+from baram.log_manager import LogManager
 
 class EC2Manager(object):
     def __init__(self):
         self.cli = boto3.client('ec2')
+        self.logger = LogManager().get_logger()
 
     def list_security_groups(self):
-        '''
+        """
         Describes the specified security groups or all of your security groups.
 
         :return: SecurityGroups
-        '''
+        """
         return self.cli.describe_security_groups()['SecurityGroups']
 
+    def list_instances(self):
+        """
+        Describes all existing instances.
+
+        :return:
+        """
+        instances = self.cli.describe_instances()['Reservations']
+        return [instance['Instances'][0] for instance in instances]
```

Comment:
What if there's no instance? I think it's possible depending on each AWS environment.

Reply:
Should be more defensive. I'll apply `try & except` .